changelog.md

```diff
@@ -6,6 +6,14 @@
 - bpk-component-popover:
  - On mobile screen sizes, popovers now have a margin to prevent them from filling the entire width of their container.
 
+**Fixed:**
+- bpk-component-popover:
+- bpk-react-utils:
+  - `target` prop type is now correctly `oneOf([function, node])` instead of `oneOf([function, element])`
```

Comment:
Flow finding bugs for us?

Reply:
Yeah!! I tried to use Element and flow was having none of it which made me realise I had use the wrong prop-type - strange that the prop-type didn't blow up in the first place 🤔 .

Reply:
👍